packages/react-cosmos-component-playground/src/components/FixtureList/index.js

```diff
@@ -90,23 +58,29 @@ export default class FixtureList extends Component {
     }
   };
 
-  onChange = e => {
-    this.setState({
-      searchText: e.target.value
-    });
+  onSearchChange = e => {
```

Comment:
You have to update line 28 to `window.__changePlaygroundSearch = this.onSearchChange;` for the Cypress smoke tests to pass

Reply:
Done